app/ui/templates/studio.html

```
@@ -0,0 +1,379 @@
+        </div>
+
+        <div class="action-btns" id="action-btns" style="display:flex;flex-direction:column;gap:6px;">
+          {% if talk.status == 'needs_work' %}
+            <button class="btn btn-primary w-full" id="btn-approve" onclick="approveTalk({{ talk.id }})">
+              Re-cut Timestamps & Regenerate Preview
+            </button>
+            <button class="btn btn-danger w-full" id="btn-reject" onclick="rejectTalk({{ talk.id }})">
+              Reject Talk
+            </button>
+          {% elif talk.status in ['pending_approval', 'waiting_for_files', 'pending_bounds'] %}
+            <button class="btn btn-success w-full" id="btn-approve" onclick="approveTalk({{ talk.id }})">
```

HTMLHint (1.9.2)

[error] The id value [ btn-approve ] must be unique.

(id-unique)